src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -59,4 +60,12 @@ public LineController(final LineService lineService) {
 		lineService.deleteLine(id);
 		return ResponseEntity.ok().build();
 	}
+
+	@PostMapping("/{id}/sections")
+	public ResponseEntity<LineResponse> addSection(
+		@PathVariable final Long id,
+		@RequestBody final SectionRequest sectionRequest
```

Comment:
`Request body`로 들어오는 요청 값에 대해서도 일종의 검증을 할 수 있는데요 😄 
```suggestion
		@RequestBody @Valid final SectionRequest sectionRequest
```
`@Valid` 어노테이션을 붙이고 `SenctionRequest`에 적절한 어노테이션을 붙이면 `body`에 대한 일차 검증이 가능합니다 😄 
이 부분은 참고만 해주시면 감사하겠습니다 😄 

도움 되실 만한 내용 공유 드립니다 
> https://meetup.toast.com/posts/223

Reply:
조언 감사드립니다 😃